### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by nsantiagoblair**
Small style fixes after layout

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -190,13 +226,13 @@ nav .nav-list {
 }
```
Comment (ailindoyle) [inline styles.css]: extra line break

Comment (ailindoyle) [inline styles.css]: extra line break

File: styles.css [hunk 2]
```diff
@@ -236,6 +272,10 @@ nav .nav-list {
```
Comment (ailindoyle) [inline styles.css]: extra line break

File: styles.css [hunk 3]
```diff
@@ -236,6 +272,10 @@ nav .nav-list {
 
 
 @media screen and (max-width: 1280px) {
+    nav {
+        padding: 0 150px
+    }
+
     section {
```
Comment (ailindoyle) [inline styles.css]: line break needed

File: styles.css [hunk 4]
```diff
@@ -245,101 +285,126 @@ nav .nav-list {
...
     }
 }
 
 @media screen and (max-width: 1024px) {
 
     nav {
-        padding: 0 100px
+        padding: 0 80px
     }
     section {
```
Comment (ailindoyle) [inline styles.css]: line break needed

File: styles.css [hunk 5]
```diff
@@ -245,101 +285,126 @@ nav .nav-list {
...
         margin: 0 0 0 0;
     }
 }
 
 
 @media screen and (max-width: 860px) {
+    nav {
+        text-align: center;
+    }
+    nav .site-logo {
```
Comment (ailindoyle) [inline styles.css]: line break needed

File: styles.css [hunk 6]
```diff
@@ -245,101 +285,126 @@ nav .nav-list {
...
 
-    .contact .map {
+    #contact .map {
         margin: 0 0 0 0;
     }
 }
 
 
 @media screen and (max-width: 860px) {
+    nav {
```
Comment (ailindoyle) [inline styles.css]: line break needed

File: styles.css [hunk 7]
```diff
@@ -24,7 +24,7 @@ section {
 a {
     text-decoration: none;
     color: white;
-    font-family:  'Permanent Marker', cursive;
+    font-family: 'Permanent Marker', cursive;
```
Comment (marty-crane) [inline styles.css]: White space

File: styles.css [hunk 8]
```diff
@@ -73,24 +75,47 @@ nav .nav-list {
...
 }
 
+
 .nav-list a:hover {
-    text-decoration: underline white;
+    text-shadow: 5px 5px 5px gray, -5px -5px 5px gray,
+                -5px 5px 5px gray, 5px -5px 5px gray;
 }
 
+.nav-list a:hover { text-decoration: underline }
```
Comment (marty-crane) [inline styles.css]: Should text-decoration be on a new line?

File: styles.css [hunk 9]
```diff
@@ -73,24 +75,47 @@ nav .nav-list {
...
+
+.home .bg-video {
+    position: absolute;
+    top: 0;
+    left: 0;
+    object-fit: cover;
+    width: 100%;
+    height: 100vh;
+    margin: 0;
+    z-index: -5;
```
Comment (marty-crane) [inline styles.css]: z-index negative - could change rest of code to fix......

File: styles.css [hunk 10]
```diff
@@ -245,101 +314,142 @@ nav .nav-list {
...
     }
+
+    #contact {
+        background-image: url('assets/map-mobile.png');
+    }
+
+    #contact .details {
+        margin-left: 50px;
+    }
+
```
Comment (marty-crane) [inline styles.css]: Line breaks not needed

**Pull request merged**